### DISCUSSION TITLE
Building docker image from source

### DISCUSSION THREAD

**Discussion by sloethen** [OrchardCMS/OrchardCore#16586] [Q&A]
I am customizing the source (mainly for my education, and to create a custom theme) and wish to deploy the resulting product as a docker image to my Synology NAS.  My assumption was the docker file at the root would be all I need.  

When I do a docker build --tag <some tag> I get a error on the dotnet publish line, telling me targetedframework is ''.  The call shows net8.0 in the call.  The web project file is default, but shows the $(targetedcommonframeworks) and has .net8.0 selected (all of that was out of box.

I tried it on my modified source, on a unmodified clone of the source, and on 2 different machines.  Is there something I am missing?

**Comment by Piedone** [comment]
It was you who posted to Discord as well about this, right? https://discord.com/channels/551136772243980291/551137154705653775/1275100630343745577

**Reply by sloethen** [reply]
Yes, it was I

**Reply by sloethen** [reply]
Is this or Discord better?

**Reply by Piedone** [reply]
For longer questions this is better. But in any case, please either post to a single place, or link one from the other, so we don't duplicate efforts and help other people looking for answers.
